utils/childAnswerParser.js

```diff
@@ -0,0 +1,9 @@
+const { endsWith } = require("lodash/fp");
+
+module.exports = id => {
+  if (endsWith("from", id)) {
+    return "primary";
```

Comment:
Usually, best to use constant variables rather than strings in these scenarios.  It's ok in this instance as it sounds like we will be re-visiting this area when we build additional composite answers.